src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -222,13 +224,13 @@ def prop_dict(self):
             'group': self._resource_group
         }
 
-    def __init__(self, cmd, payload, operation):
+    def __init__(self, cmd, payload, operation, model_path=None):
```

Comment:
model_path for extensions tends to be {extension}.{extension subpackage}.vendored_sdks.models instead of that specified in the model docstring. This lets us override the model_path.